is_core/generic_views/form_views.py

```diff
@@ -58,11 +60,14 @@ def get_success_url(self, obj):
     def get_has_file_field(self, form, **kwargs):
         return formset_has_file_field(form)
 
+    def is_readonly(self):
+        return not self._check_call('post')
```

Comment:
Proč přímo nekontroluješ to právo na post?

Reply:
jak to myslíš?

Reply:
Jako přímo kontrolovat to právo.
`self.permission.has_permission('post', self.request, self)`

Reply:
protože tam pak chybí objekt a já ho potřebuju. Vyřešil jsem to jinak